storm-client/src/jvm/org/apache/storm/daemon/supervisor/ClientSupervisorUtils.java

```diff
@@ -64,7 +64,7 @@ public static int processLauncherAndWait(Map<String, Object> conf, String user,
                                              final Map<String, String> environment, final String logPreFix)
         throws IOException {
         int ret = 0;
-        Process process = processLauncher(conf, user, null, args, environment, logPreFix, null, null);
+        Process process = processLauncher(conf, user, null, args, environment, null, null, null);
```

Comment:
I agree this is a good change.

It isn't obvious to me that adding a prefix should cause the threading behavior.  I'd rather see an explicit API with either a name change or a thread parameter rather than relying on logPrefix.

Reply:
Yes I totally agree. Filed a separate jira: https://issues.apache.org/jira/browse/STORM-3389

Reply:
Thanks